src/main/java/org/prebid/server/deals/targeting/interpret/IntersectsStrings.java

```diff
@@ -16,7 +16,8 @@ public IntersectsStrings(TargetingCategory category, List<String> values) {
 
     @Override
     public List<String> lookupActualValues(RequestContext context) {
-        return toLowerCase(context.lookupStrings(category));
+        final List<String> values = context.lookupStrings(category);
+        return values != null ? toLowerCase(values) : null;
```

Comment:
Why not return `Collections.emptyList()` here and leave previous implementation of `matches`?

Reply:
NULL is more optimized. If look forward in `org.prebid.server.deals.targeting.interpret.Intersects#matches` method - the call of `Collections.disjoint(..)` is not required. Another though - `lookupActualValues(..)` method returns values in request, so empty list can real value.